tests/Integration/Behaviour/Features/Context/Domain/OrderRefundFeatureContext.php

```diff
@@ -27,22 +27,35 @@
 namespace Tests\Integration\Behaviour\Features\Context\Domain;
 
 use Behat\Gherkin\Node\TableNode;
+use Configuration;
 use Order;
 use OrderSlip;
 use PHPUnit\Framework\Assert as Assert;
 use PrestaShop\PrestaShop\Core\Domain\Order\Command\IssuePartialRefundCommand;
+use PrestaShop\PrestaShop\Core\Domain\Order\Command\IssueReturnProductCommand;
+use PrestaShop\PrestaShop\Core\Domain\Order\Command\IssueStandardRefundCommand;
+use PrestaShop\PrestaShop\Core\Domain\Order\Exception\InvalidOrderStateException;
 use PrestaShop\PrestaShop\Core\Domain\Order\Exception\InvalidRefundException;
 use PrestaShop\PrestaShop\Core\Domain\Order\Exception\OrderException;
+use PrestaShop\PrestaShop\Core\Domain\Order\Exception\ReturnProductDisabledException;
 use PrestaShop\PrestaShop\Core\Domain\Order\Query\GetOrderForViewing;
 use PrestaShop\PrestaShop\Core\Domain\Order\QueryResult\OrderForViewing;
 use PrestaShop\PrestaShop\Core\Domain\Order\QueryResult\OrderProductForViewing;
-use PrestaShop\PrestaShop\Core\Domain\Order\ValueObject\OrderId;
 use PrestaShop\PrestaShop\Core\Domain\Order\VoucherRefundType;
 use RuntimeException;
 use Tests\Integration\Behaviour\Features\Context\SharedStorage;
 
 class OrderRefundFeatureContext extends AbstractDomainFeatureContext
 {
+    /**
+     * @BeforeScenario
+     */
+    public function before()
+    {
+        // Merchandise return is disabled by default, use enabledReturnProduct() to enable it
+        Configuration::set('PS_ORDER_RETURN', 0);
+    }
```

Comment:
Is there an `after` for reset the configuration to its default value ?

Reply:
Actually it should be 0 already with the default sql dump But I change this value with a Behat statement to enable it, and in following scenarii it was not reset to 0 I don't understand exactly when the dump is reinserted in the database..
This might be worth resetting on an after indeed, for now it doesn't create side effect on the behat suite so maybe we can fix this in the last PR of cancel product (to avoid restarting the whole review + QA process)

@matthieu-rolland could you integrate an `after` in your PR https://github.com/PrestaShop/PrestaShop/pull/17149 please?